float64[2] sj_p
float64[2] sj_v
float64[2] sj_acc
float64[2] sj_dec
